test/test_helper/common.bash

```diff
@@ -1,7 +1,7 @@
 load 'test_helper/bats-support/load'
 load 'test_helper/bats-assert/load'
 
-NAME=tvial/docker-mailserver:testing
+# NAME=tvial/docker-mailserver:testing
```

Comment:
Since you did not comment on this removal:
1. Why did you just comment out rather than delete?
2. this is actually used in the *.bats files, where does it come from now?

Reply:
Exactly was I was out for because I was not 100% sure whether everything was correct. All other changes were less invasive:) I fixed this issue with a new commit.